tool/tsh/common/tsh.go

```diff
@@ -904,24 +906,28 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 
 	proxyApp := proxy.Command("app", "Start local TLS proxy for app connection when using Teleport in single-port mode.")
 	proxyApp.Arg("app", "The name of the application to start local proxy for").Required().StringVar(&cf.AppName)
-	proxyApp.Flag("port", "Specifies the source port used by by the proxy app listener").Short('p').StringVar(&cf.LocalProxyPort)
+	proxyApp.Flag("port", "Specifies the source port used by the proxy app listener").Short('p').StringVar(&cf.LocalProxyPort)
+	proxyApp.Flag("interface", "Specifies the listen interface used by the proxy app listener").Default("localhost").StringVar(&cf.LocalProxyInterface)
```

Comment:
@zmb3 that seems appropriate however do you agree that using `--bind-addr` also replaces `--port`? As in, one would then use `--bind-addr 127.0.0.1:9000` instead of `--port 9000 --interface 127.0.0.1`.